angular/README.md

```diff
@@ -1,17 +1,83 @@
 ## Disclaimer
+
 ---
-**This started is work-in-progress. Once the widget is complete, this disclaimer will be removed. Thank you for your patience.**
+
+**This starter is work-in-progress. Once the widget is complete, this disclaimer will be removed. Thank you for your patience.**
 
 ---
-# WCC Widget Starter: Angular
-## Development
 
-Setting up your development environment:
+# WxCC Widget Starter - Angular
 
-1. Clone this repo
-2. Run `yarn` from the root of the repo.
-3. Run `yarn start` to start the playground app.
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.2.11.
 
-## Disclaimer
+This widget starter can be run on `localhost` to be reviewed. In case you (the developer) are using this widget starter as a foundation of your custom widget, running the sandbox environment on `localhost` would be a perfect way to build your widget's logic and interface.
+
```

Comment:
How did we test this widget?
Did we test following scenario?
1. place this widget in the custom page of the desktop
2. place this widget in the aux-panel of the desktop
3. place this widget as header of the desktop
verify if widget getting updated data fromSTORE

Reply:
This widget was tested by packaging it and placing it into a custom page in desktop, where it is getting updated data from STORE. 
The other 2 scenarios can be verified